app/jobs/background_render/inactive_client_report_job.rb

```diff
@@ -15,7 +17,7 @@ def render_html(filters:, user_id:, page:)
     current_user = User.find(user_id)
     @filter = ::Filters::FilterBase.new(user_id: user_id).set_from_params(JSON.parse(filters).with_indifferent_access)
     set_report
-    @pagy, @clients = pagy(@report.clients.order(:last_name, :first_name), page: page, params: @filter.for_params)
+    @pagy, @clients = pagy_array(@report.clients.sort_by { |c| [c.last_name, c.first_name] }, page: page, params: @filter.for_params)
```

Comment:
I'm still confused as to how this is helping.  This is going to pull back all `@report.clients` every time and then sort them, whereas before it was pulling back only the selected page of 25.